Fix tests in v2.3.0

## What went wrong? 😲

```
---- refresh_dates::tests::all_fields_are_present stdout ----
thread 'refresh_dates::tests::all_fields_are_present' panicked at cli/src/refresh_dates.rs:706:13:
assertion `left == right` failed: Field TableNameAndFieldRow {
    table_name: "invoice",
    column_name: "backdated_datetime",
} is in schema but not in get_timestamp_fields
  left: 1
 right: 0
```

This is my bad :(

```
error[E0063]: missing field `store_id` in initializer of `program_enrolment_row::ProgramEnrolmentRow`
   --> repository/src/db_diesel/patient.rs:469:26
    |
469 |             .upsert_one(&ProgramEnrolmentRow {
    |                          ^^^^^^^^^^^^^^^^^^^ missing `store_id`

error[E0063]: missing field `store_id` in initializer of `program_enrolment_row::ProgramEnrolmentRow`
   --> repository/src/db_diesel/patient.rs:577:26
    |
577 |             .upsert_one(&ProgramEnrolmentRow {
    |                          ^^^^^^^^^^^^^^^^^^^ missing `store_id`
```

This one I should have run tests before approving a PR 😢 

## Expected behaviour 🤔

Tests should pass (even on postgres!)

## How to Reproduce 🔨

Run tests

## Your environment 🌱

<!-- e.g. 1.2.3 -->

- Open mSupply Version:
- Legacy mSupply Central Server Version:
<!-- e.g. android, browser (extra points if you tell us which one), desktop (windows), desktop (macOS), server (windows) -->
- Platform:
<!-- PostgreSQL or SQLite3 -->
- Database type:

--- a/server/cli/src/refresh_dates.rs
+++ b/server/cli/src/refresh_dates.rs
@@ -51,6 +51,7 @@ fn get_timestamp_fields() -> Vec<TableAndFieldName> {
         ("rnr_form", "created_datetime"),
         ("rnr_form", "finalised_datetime"),
         ("vaccination", "created_datetime"),
+        ("invoice", "backdated_datetime"),
     ]
     .iter()
     .map(|(table_name, field_name)| TableAndFieldName {

--- a/server/repository/src/db_diesel/patient.rs
+++ b/server/repository/src/db_diesel/patient.rs
@@ -475,6 +475,7 @@ mod tests {
                 enrolment_datetime: Utc::now().naive_utc(),
                 program_enrolment_id: Some("program_enrolment_id".to_string()),
                 status: Some("Active".to_string()),
+                store_id: None,
             })
             .unwrap();
 
@@ -583,6 +584,7 @@ mod tests {
                 enrolment_datetime: Utc::now().naive_utc(),
                 program_enrolment_id: Some("program_enrolment_id".to_string()),
                 status: Some("Active".to_string()),
+                store_id: None,
             })
             .unwrap();
         let repo = PatientRepository::new(&connection);

